### ISSUE TITLE
Camera selection?

### ISSUE THREAD

**Issue by englishm** [nolenroyalty/paccam#1]
Sometimes the default device is not the one you want to use (e.g. an OBS virtual camera that isn't running)

**Comment by nolenroyalty** [comment]
oh, ugh, dying for all the browsers to implement built in camera selection (chrome just did this! it's nice!)

**Comment by nolenroyalty** [comment]
yeah this is annoying i'll try to add something

**Comment by englishm** [comment]
Oh hey, you're right! Thanks!

I had _thought_ Chrome had this now, too, but couldn't find it at first. I had just lost track of how to get to it.

It looks like you have to navigate into settings now to change the default camera.

In case it's helpful for anyone else, here are some screenshots.

<img width="351" alt="Screenshot 2024-11-01 at 12 43 05 PM" src="https://github.com/user-attachments/assets/d46ccc8f-615c-492d-9914-b1a36ab998b4">

Clicking Manage takes you to <chrome://settings/content/camera>

<img width="720" alt="Screenshot 2024-11-01 at 12 43 49 PM" src="https://github.com/user-attachments/assets/fa1fdb48-9a0a-46d2-945b-744432fbe73f">

**Comment by nolenroyalty** [comment]
oh the newest version of chrome (on mac) prompted me when i enabled webcam for paccam for the first time and let me choose on the page (I think? or maybe that was another browser i was testing on...).

regardless, having a little widget to let you do this till all browsers do the useful thing in an easy way would be nice

**Comment by englishm** [comment]
Firefox actually has an even more discoverable camera selection feature upon first granting permission, so maybe we can just close this?

<img width="488" alt="Screenshot 2024-11-01 at 12 49 43 PM" src="https://github.com/user-attachments/assets/c0bc163d-e28b-4601-a938-fb6539460321">

**Comment by englishm** [comment]
> oh the newest version of chrome (on mac) prompted me when i enabled webcam for paccam for the first time and let me choose on the page (I think? or maybe that was another browser i was testing on...).

Yeah, I thought I remembered Chrome working this way at one point, too, but maybe I also got it mixed up with Firefox's flow. 

I just reset permissions and tested again and didn't get a choice of cameras when granting permission in Chrome.

<img width="532" alt="Screenshot 2024-11-01 at 12 52 45 PM" src="https://github.com/user-attachments/assets/d792e7ee-e542-48f2-979b-9e73b9bc5809">

Either way, that's what made me think it might not be possible to change cameras in Chrome without the site supporting device selection.

> regardless, having a little widget to let you do this till all browsers do the useful thing in an easy way would be nice

It might help avoid some confusion at least!

**Comment by englishm** [comment]
P.S. Now that I've been able to actually play it a bit I can say that this game is just as fantastic as I hoped it would be reading about the concept. :D
